plugins/dahua.rb

```diff
@@ -0,0 +1,17 @@
+Plugin.define do
+  name "Dahua IP Cameras"
+  authors [
+    "Ostorlab",
+  ]
+  version "0.1"
+  description "Dahua network IP cameras are the preferred brand among most security and IT professionals due to the low price point, high reliability, and wide compatibility with most NVRs and VMS platforms. "
+  website "https://www.dahuasecurity.com/"
+
+  matches [
+    {
+      :search => "body",
+      :regexp => /<script\s+src="jsBase\/lib\/jquery\.js\?version=@WebVersion@"><\/script>/,
```

Comment:
Are you sure no other services or website will be using this js in them 
```suggestion
      :regexp => /<script\s+src="jsBase\/lib\/jquery\.js\?version=@WebVersion@"><\/script>/,
```
this can result in FP

Reply:
that's the query used to search for vulnerable instances in `fofa`: `body="jsBase/lib/jquery.js?version=@WebVersion@"`